BeerCatalogFullstack/FrontEnd/src/helpers/requestHelper.js

```diff
@@ -0,0 +1,71 @@
+function get(url) {
```

Comment:
You have big amount of reusable logic here. Try to create some function with common logic and specific functions like `get`, `post`, etc will path params there.